access-functions.php

```diff
@@ -295,6 +295,23 @@ function( \WPGraphQL\Registry\TypeRegistry $type_registry ) use ( $type_name, $f
 	);
 }
 
+/**
+ * Renames GraphQL fields.
+ *
+ * @param string $typename      Type being renamed.
+ * @param string $new_typename  New typename.
+ *
+ * @return void
+ */
+function rename_graphql_type( string $typename, string $new_typename ) {
+	add_filter(
+		"graphql_typename_{$typename}",
```

Comment:
along with my other comment, I think this should be `graphql_type_name`, then the callback should include something like: 

```php
function( $existing_typename ) use ( $typename, $new_typename ) {
  if ( $existing_typename !== $typename ) {
    return $new_typename;
  }
  return $existing_typename;
}
```